cmd/gitops-server/main.go

```diff
@@ -71,11 +72,12 @@ func NewAPIServerCommand() *cobra.Command {
 			}
 
 			profileWatcher, err := watcher.NewWatcher(watcher.Options{
-				KubeClient:         rawClient,
-				Cache:              profileCache,
-				MetricsBindAddress: metricsBindAddress,
-				HealthzBindAddress: healthzBindAddress,
-				WatcherPort:        watcherPort,
+				KubeClient:                    rawClient,
+				Cache:                         profileCache,
+				MetricsBindAddress:            metricsBindAddress,
+				HealthzBindAddress:            healthzBindAddress,
+				NotificationControllerAddress: notificationBindAddress,
+				WatcherPort:                   watcherPort,
```

Comment:
Why are these not cmd-line flags?

Reply:
They are, but not in this section which doesn't have command line flags.